frontend/tsconfig.json

```diff
@@ -7,13 +7,12 @@
     "declaration": false,
     "moduleResolution": "node",
     "strict": false,
+    "esModuleInterop": true,
```

Comment:
It might be a good idea to remove `import * ` from the code.
In a separate PR.

Reply:
Probably a good idea at some point. In the frontend code, there are no instances of `import *` for 3rd party deps, only modules defined locally.